src/site/xdoc/index.xml

```diff
@@ -15,346 +15,122 @@
-              </p>
-            </div>
-            <p><img src="logo-call/james-logo-5-avatar.png"/></p>
-          </subsection>
-
-          <subsection name="Proposal 6">
-            <div id="james-logo-6-preview">
-              <p><img src="logo-call/james-logo-6.png"/></p>
-              <p>
-                <a href="#">Click to change the upper left logo on this page and preview this proposal.</a>
-              </p>
-            </div>
-            <p><img src="logo-call/james-logo-6-avatar.png"/></p>
-          </subsection>
-
-          <subsection name="Proposal 7">
-            <div id="james-logo-7-preview">
-              <p><img src="logo-call/james-logo-7.png"/></p>
-              <p>
-                <a href="#">Click to change the upper left logo on this page and preview this proposal.</a>
-              </p>
-            </div>
-          </subsection>
-
-          <subsection name="Proposal 8">
-            <div id="james-logo-8-preview">
-              <p><img src="logo-call/james-logo-8.png"/></p>
-              <p>
-                <a href="#">Click to change the upper left logo on this page and preview this proposal.</a>
-              </p>
-            </div>
-            <p><img src="logo-call/james-logo-8-avatar.png"/></p>
-          </subsection>
-
-          <subsection name="Proposal 9">
-            <div id="james-logo-9-preview">
-              <p><img src="logo-call/james-logo-9.png"/></p>
-              <p>
-                <a href="#">Click to change the upper left logo on this page and preview this proposal.</a>
-              </p>
-            </div>
-          </subsection>
-
-          <subsection name="Proposal 10">
-            <div id="james-logo-10-preview">
-              <p><img src="logo-call/james-logo-10.png"/></p>
-              <p>
-                <a href="#">Click to change the upper left logo on this page and preview this proposal.</a>
-              </p>
-            </div>
-          </subsection>
-
-        </div>
-
-<!-- 
-        <div id="tabs-5">
-  
-              <object width="480" height="385">
-                <param name="movie" value="http://www.youtube.com/v/0Mx7qy7i3Hs?fs=1&amp;hl=en_US"></param>
-                <param name="allowFullScreen" value="true"></param><param name="allowscriptaccess" value="always"></param>
-                <embed src="http://www.youtube.com/v/0Mx7qy7i3Hs?fs=1&amp;hl=en_US" type="application/x-shockwave-flash" allowscriptaccess="always" allowfullscreen="true" width="480" height="385">
-                </embed>
-              </object>
-  
-        </div>
--->
+
+          <subsection name="Software Components">
+
+              <p>Developers looking for a modular mail platform on which to build can look
+                 at the modules and libraries used to compose James Server 3.0.</p>
+
+              <p><a href='http://james.apache.org/imap/'>Apache James IMAP</a> provides a flexible codec for IMAP,
```

Comment:
Apache James IMAP is now part of protocols, and is not separated from it. Merging IMAP into protocols have been done before we starte working on it. Thus, it may be nice not to document it as a software toplevel project but a protocols component.